apps/swirl-docs/src/components/Tokens/TokenItem.tsx

```diff
@@ -7,17 +24,42 @@ import {
   isZindexToken,
 } from "@swirl/lib/tokens/src/utils";
 import classNames from "classnames";
+import { useState } from "react";
+import CopyToClipboard from "react-copy-to-clipboard";
 import TokenPreview from "./TokenPreview";
 
 export type TokenItemProps = {
   token: Token;
 };
 
 const TokenItem = ({ token }: TokenItemProps) => {
+  const [open, setOpen] = useState(false);
+  const [isCopied, setIsCopied] = useState(false);
+
+  const { x, y, strategy, refs, context } = useFloating({
```

Comment:
**suggestion:** Could we add a short delay before showing the tooltip?

Reply:
yes. i would start with 500ms?

Reply:
Our Swirl tooltips have a default of 300ms.